src/map/packet_system.cpp

```diff
@@ -5349,6 +5349,8 @@ void SmallPacket0x0FA(map_session_data_t* session, CCharEntity* PChar, CBasicPac
             PChar->getStorage(LOC_STORAGE)->AddBuff(PItem->getStorage());
 
             PChar->pushPacket(new CInventorySizePacket(PChar));
```

Comment:
if you're going to go the route of char_vars, you can eliminate two steps and unnecessary bindings by just having the core do that work here:
```
            const char* itemQuery = "INSERT INTO char_vars(charid, varname, value) VALUES (%u,'[MH]%s',%u);";
            Sql_Query(SqlHandle, itemQuery, PChar->id, PItem->getName(), CVanaTime::getInstance()->getVanaTime() + 1009810800);
```

Reply:
We may have other future things we want to trigger via furniture changes without recompiling. This was discussed on discord already.

Reply:
@ibm2431 looks like github also puts a priority on "review" comments so they appear first instead of in chronological order.

Reply:
@TeoTwawki That was an impressive bit of magic, wasn't it. I think that may be because I had it queued in the "review" and was still looking through stuff when you chimed in, and Github wanted to keep it in that order to show at the time I added the comment, I "hadn't seen" yours yet (even though I did in another tab).